packages/nestjs-firebase-admin/README.md

```diff
@@ -4,6 +4,10 @@
 
 <h1 align="center">Nestjs Firebase Admin Module</h1>
 
+<p align="center">
+  <img src="https://github.com/cuaklabs/nestjs-meow/workflows/ci/badge.svg" alt="Build status"/>
+</p>
+
```

Comment:
We could add the codecov badge [![codecov](https://codecov.io/gh/cuaklabs/nestjs-meow/branch/master/graph/badge.svg?token=SA47S3AN5P)](https://codecov.io/gh/cuaklabs/nestjs-meow)